test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -1023,14 +1023,9 @@ private static void Describe(HttpResponse res, AuthenticateContext result)
             {
                 xml.Add(result.Properties.Select(extra => new XElement("extra", new XAttribute("type", extra.Key), new XAttribute("value", extra.Value))));
             }
-            using (var memory = new MemoryStream())
```

Comment:
It did not appear that there was support for XmlTextWriter in dnxcore50.  I needed to replace it with something that was compatible.  Is there a better way?  I wasn't sure about these.  The same code is duplicated in multiple places, but that is a whole other issue.